attribute-masking-service/src/main/resources/application-static.yaml

```diff
@@ -15,10 +15,24 @@
 spring:
   profiles:
     include:
-    - h2
```

Comment:
If this is a Spring profile that we have removed, the corresponding application-h2.yaml file could be removed.  Also I don't see a yaml file for static-config. Should this be removed from the file as well?

Reply:
This was for giving the profiles a dependency-like mechanism.
In `static` profile, we automatically enable `h2` (or used to) and `static-config` (the stuff below the `---`).
`h2` is still used for tests because it's faster and easier. `static-config` is used by everything that uses static